common/config/rush/npm-shrinkwrap.json

```diff
@@ -5,27 +5,73 @@
+      "dependencies": {
+        "typescript": {
+          "version": "2.2.2",
+          "from": "typescript@>=2.2.2 <2.3.0",
+          "resolved": "https://registry.npmjs.org/typescript/-/typescript-2.2.2.tgz"
+        }
+      }
     },
     "@microsoft/gulp-core-build": {
-      "version": "2.5.6",
-      "from": "@microsoft/gulp-core-build@>=2.4.3 <3.0.0",
-      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build/-/gulp-core-build-2.5.6.tgz"
+      "version": "2.7.0",
+      "from": "@microsoft/gulp-core-build@>=2.5.6 <3.0.0",
+      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build/-/gulp-core-build-2.7.0.tgz"
     },
     "@microsoft/gulp-core-build-mocha": {
-      "version": "2.0.3",
-      "from": "@microsoft/gulp-core-build-mocha@>=2.0.2 <3.0.0",
-      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build-mocha/-/gulp-core-build-mocha-2.0.3.tgz"
+      "version": "2.1.0",
+      "from": "@microsoft/gulp-core-build-mocha@>=2.1.0 <3.0.0",
+      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build-mocha/-/gulp-core-build-mocha-2.1.0.tgz"
     },
     "@microsoft/gulp-core-build-typescript": {
-      "version": "3.1.5",
-      "from": "@microsoft/gulp-core-build-typescript@>=3.0.3 <4.0.0",
-      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build-typescript/-/gulp-core-build-typescript-3.1.5.tgz"
+      "version": "3.2.0",
+      "from": "@microsoft/gulp-core-build-typescript@>=3.2.0 <4.0.0",
+      "resolved": "https://registry.npmjs.org/@microsoft/gulp-core-build-typescript/-/gulp-core-build-typescript-3.2.0.tgz",
+      "dependencies": {
+        "diff": {
+          "version": "3.3.0",
+          "from": "diff@>=3.0.1 <4.0.0",
+          "resolved": "https://registry.npmjs.org/diff/-/diff-3.3.0.tgz"
+        },
+        "findup-sync": {
+          "version": "0.3.0",
+          "from": "findup-sync@>=0.3.0 <0.4.0",
+          "resolved": "https://registry.npmjs.org/findup-sync/-/findup-sync-0.3.0.tgz",
+          "dependencies": {
+            "glob": {
+              "version": "5.0.15",
+              "from": "glob@>=5.0.0 <5.1.0",
+              "resolved": "https://registry.npmjs.org/glob/-/glob-5.0.15.tgz"
+            }
+          }
+        },
+        "glob": {
+          "version": "7.1.2",
+          "from": "glob@>=7.1.1 <8.0.0",
+          "resolved": "https://registry.npmjs.org/glob/-/glob-7.1.2.tgz"
+        },
+        "tslint": {
+          "version": "4.5.1",
+          "from": "tslint@>=4.5.1 <4.6.0",
+          "resolved": "https://registry.npmjs.org/tslint/-/tslint-4.5.1.tgz"
+        },
+        "tslint-microsoft-contrib": {
+          "version": "4.0.1",
+          "from": "tslint-microsoft-contrib@>=4.0.0 <4.1.0",
+          "resolved": "https://registry.npmjs.org/tslint-microsoft-contrib/-/tslint-microsoft-contrib-4.0.1.tgz"
+        },
+        "typescript": {
```

Comment:
Yup. I'm going to do a followup PR as soon as this publishes.

---
In reply to: [129394536](https://github.com/Microsoft/web-build-tools/pull/276#discussion_r129394536) [](ancestors = 129394536)